File: Typescript/Practice/26072022/medium/medium_tileTeamworkTactics.py
#!/usr/bin/env python

def possible_bonus(pos1, pos2):
  if (pos1 < pos2) and (pos2 - pos1 <= 6):
    return True
  return False

print(possible_bonus(3, 7))
print(possible_bonus(1, 9))
print(possible_bonus(5, 3))

File: Typescript/Practice/medium/medium_tileTeamworkTactics.ts
#!/usr/bin/env npx ts-node

function possibleBonus (pos1: number, pos2:number): boolean {
  let result: boolean = false;
  if (pos1 < pos2) {
    if (pos2 - pos1 <= 6) {
      result = true;
    };
  };
  return result
}

console.log(possibleBonus(3, 7));
console.log(possibleBonus(1, 9));
console.log(possibleBonus(5, 3));
